themes/default/content/blog/announcing-kubernetes-helm-release-resource-preview/index.md

```diff
@@ -0,0 +1,305 @@
+---
+title: Announcing Preview for Helm Release Resource for Kubernetes
+h1: "Announcing Preview for Helm Release Resource for Kubernetes"
+date: "2021-09-07"
```

Comment:
```suggestion
date: "2021-09-15"
```

Reply:
We can release the blog as soon as we are ready, no need to wait until 15th

Reply:
Updated to today.